galley/testdata/validation/dataset/config-v1alpha2-QuotaSpec-valid.yaml

```diff
@@ -9,5 +9,5 @@ spec:
         api.operation:
           exact: getProducts
     quotas:
-    - charge: "1"
+    - charge: 1
```

Comment:
do you need to update dataset.gen.go?

Reply:
Because it is testing with `"1"` still, see the yaml it applies: https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio/17900/integ-galley-k8s-tests_istio/2272/artifacts/galley-test-843dc40cb84e47d0947/_suite_context/env-kube-686732646/istio-kube-accessor-106593160/accessor_240605183